.github/workflows/github_ci.yml

```diff
@@ -108,3 +117,31 @@ jobs:
           distribution: 'temurin'
       - name: Checkstyle
         run: ./gradlew checkstyleMain checkstyleTest
+  frontend_changed:
+    name: Frontend (matrix build)
+    needs: [build, check]
+    if: ${{ needs.check.outputs.frontend_changed == 'true' }}
+    strategy:
+      matrix:
+        os: [ windows-latest, macos-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout frontend
+        uses: actions/checkout@v4
+        with:
+          path: blockly/frontend
```

Comment:
wenn ich das richtig in erinnerung habe, wird das repo in diesen pfad ausgecheckt. d.h. der gesamte inhalt des dungeon-repos liegt dann unter "./blockly/frontend/". d.h. das blockly-frontend wäre dann unter "./blockly/frontend/blockly/frontend/" zu finden? d.h. dann passen die anderen pfadangaben in den nächsten schritten nicht mehr?

Reply:
Habe auf `filter` gewechselt das sollte das machen was ich wollte